source/services/test/chaiMoment.tests.ts

```diff
@@ -0,0 +1,34 @@
+import * as moment from 'moment';
+import 'moment-timezone';
+
+import './test.module';
+
+describe('chai moment', () => {
+	it('should return true if moments are equal', (): void => {
+		expect(moment('1/1/2015')).to.be.sameMoment(moment('1/1/2015'));
```

Comment:
aliases to the same thing. 
https://github.com/RenovoSolutions/TypeScript-Angular-Utilities/pull/112/commits/5eedd1a4b25068bbc01551c1169fb646ab50b91e#diff-be36213595837d60db7c25b5a338596aR21